bindings/python/iota_sdk/types/send_params.py

```diff
@@ -72,10 +72,12 @@ class CreateNativeTokenParams():
         account_id: The ID of the corresponding account.
     """
     circulating_supply: int = field(metadata=config(
-        encoder=str
+        encoder=hex,
+        decoder=lambda v: int(v, 0)
     ))
     maximum_supply: int = field(metadata=config(
-        encoder=str
+        encoder=hex,
+        decoder=lambda v: int(v, 0)
     ))
```

Comment:
I wonder if we could store the whole `field` in a helper function somewhere and just do `maximum_supply: int = hex_int` or something?

Reply:
hmm, yeah, probably